### PR TITLE
Update "CSS to Tailwind" page

### PR DESCRIPTION
**Pull request by Jackardios**
- [x] Update `css-to-tailwindcss` to stable v1.0.4 (with many bug fixes and [arbitrary properties](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) support)
- [x] Add `Enable arbitrary properties` setting
- [x] Add note about using classes instead of @apply directive
- [x] Improve fault tolerance

### CODE REVIEW

**Comment by ritz078** [comment]
@Jackardios Is this ready for review ?

**Comment by Jackardios** [comment]
@ritz078 Yes, its ready

File: pages/css-to-tailwind.tsx [hunk 1]
```diff
@@ -25,13 +26,23 @@ const Monaco = dynamic(() => import("../components/Monaco"), {
...
+  arbitraryPropertiesIsEnabled?: boolean;
 }
 
 const evalConfig = (configValue: string) =>
   eval(`const module = {}; ${configValue}; module.exports;`);
 
 const DEFAULT_POSTCSS_PLUGINS = [require("postcss-nested")];
 
+function decorateResult(result: string) {
+  return `/*
```
Comment (Jackardios) [inline pages/css-to-tailwind.tsx:38]: I think it's worth leaving this note so people don't think it's good practice to use `@apply`

**Review by ritz078 — APPROVED** [review]
Sorry for a long delay on this. LGTM. 
Thank you for your contribution 🎉

**Review by ritz078 — APPROVED** [review]

**Pull request merged**